CHANGELOG.md

```diff
@@ -5,28 +5,37 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+* New HDF5 support
 
 ## [0.1.3]
 ### Added
 * Pass save results as parameter for node
```

Comment:
maybe : pass "save results" ... 
to make it more clear that this is a flag for saving, or not saving